docs/conf/portland/2020/writing-day.rst

```diff
@@ -18,11 +18,39 @@ Your Project Here
 The Good Docs Project
 ---------------------
 
+ `The Good Docs Project <https://thegooddocsproject.dev>`_  is collating best practices for creating great docs. At Write the Docs Portland we'll be focusing on four key topics:
+
+**Templates**
+
 You've just been asked by your team to create documentation for your project. But what kind of document should you create? (An FAQ? A README?) What is supposed to be in it? How do you begin?
 
- `The Good Docs Project <https://github.com/thegooddocsproject/templates>`_ can help.
+ `The Good Docs Project's templates <https://github.com/thegooddocsproject/templates>`_ can help.
+
+We're working on a set of templates to make it easier to write documentation by suggesting the content needed to create the minimum viable set of documentation for your project. Drop by if you have new documentation to create, or if you want to try your existing docs against our checklists, or if you just want to talk about templates.
+
+Reach out to @ccromwell on WTD slack if interested.
+
+**How would you customize a style guide?**
+
+We have a draft `Style guide template <https://docs.google.com/document/d/1HxtaiayAJZvF0ZfNjLvRH3vYMvGTEki_TK8hFilQNJ0>`_ and would love feedback from tech writers. (Please use comments rather than track changes in the doc - it is easier to maintain.)
+
+Reach out to @alyssa on WTD slack if interested.
+
+**Transform templates to your wiki format**
+
+We want to be able to convert our markdown templates to your format? What tools, configuration settings, and how-tos should we have for this?
```

Comment:
This sentence doesn't make sense. Perhaps it should be "Do you want to be able to ..." ?